lib/snowbridge.ts

```diff
@@ -373,3 +386,37 @@ export function getErrorMessage(err: any) {
   console.error(message, err);
   return message;
 }
+
+export async function assetMetadata(
```

Comment:
It might be better in a utilities section.

Reply:
This file is already a sort of general snowbridge utilities. Including functions to get the bridge operational status, get transfer history, asset metadata and environment management. I think at some point it will be split out into more files but its only 400 LOC right now so I do not feel its important to do it right now.